grafana-plugin/playwright.config.ts

```diff
@@ -38,8 +41,6 @@ const config: PlaywrightTestConfig = {
   reporter: 'html',
   /* Shared settings for all the projects below. See https://playwright.dev/docs/api/class-testoptions. */
   use: {
-    storageState: './storageState.json',
```

Comment:
this has to be set explicitly in each of the `use` objects, in the `projects` array towards the bottom of this file. Otherwise, the "setup" project step will try to search for this storageStage file, and fail (because it is this step that _writes_ this file)